brainiak/searchlight/searchlight.py

```diff
@@ -24,6 +24,35 @@
 ]
 
 
+def Diamond5x5x5():
+    """Diamond3x3
```

Comment:
This one line description will cause confusion because it is inconsistent with the method name. Please clarify.